.github/workflows/prividium-e2e.yml

```diff
@@ -3,10 +3,10 @@ name: Prividium E2E Tests
 on:
   pull_request:
     paths:
-      - "packages/app/**"
-      - "packages/api/**"
-      - ".github/workflows/prividium-e2e.yml"
-      - "docker-compose-prividium.yaml"
+      - 'packages/app/**'
+      - 'packages/api/**'
+      - '.github/workflows/prividium-e2e.yml'
+      - 'docker-compose-prividium.yaml'
```

Comment:
Do we need to update `docker-compose-prividium.yaml` with the new config? It uses old env variables such as `PRIVIDIUM_PRIVATE_RPC_URL` and `PRIVIDIUM_PRIVATE_RPC_SECRET` and doesn't set `PRIVIDIUM_PERMISSIONS_API_URL`.